assets/src/edit-story/components/helpCenter/index.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import { useFeatures } from 'flagged';
+import styled, { ThemeProvider, css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { theme, themeHelpers } from '../../../design-system';
+import { Toggle } from './toggle';
+
+const Wrapper = styled.div`
+  position: absolute;
+  bottom: 16px;
+  left: 11px;
+  z-index: 10;
+
+  ${themeHelpers.mq.tablet(
+    css`
+      bottom: 24px;
+      left: 23px;
+    `
+  )}
+
+  ${themeHelpers.mq.desktop(
+    css`
+      bottom: 16px;
+      left: 31px;
+    `
+  )}
+`;
+
+export const HelpCenter = () => {
+  const { enableQuickTips } = useFeatures();
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    enableQuickTips && (
+      <ThemeProvider theme={theme}>
```

Comment:
Add `<ThemeGlobals.OverrideFocusOutline />` here below the ThemeProvider and you won't need to reset the button specifically, the focus will be handled via theme. The idea for this was to mimic the existing structure of global styles at the app root but since this is isolated, we can just add it here and not have to keep adding resets to everything individually.